src/api/v3/pages.py

```diff
@@ -1,6 +1,6 @@
 from django.http import JsonResponse
 
-from cms.models import Region
+from cms.models import Region, Page
```

Comment:
```suggestion
from cms.models import Region
```
remove unused import